src/dodal/beamlines/i03.py

```diff
@@ -149,6 +149,7 @@ def oav(wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False) ->
         "",
         wait_for_connection,
         fake_with_ophyd_sim,
+        params=OAVConfigParams,
```

Comment:
Must: This is just passing in a reference to the class, not an instance. Same with everywhere else we're instantiating it